lib/command/build_image.rb

```diff
@@ -11,18 +11,30 @@ class BuildImage < Base
     LONG_DESCRIPTION = <<~DESC
       - Builds and pushes the image to Control Plane
       - Automatically assigns image numbers, e.g., `app:1`, `app:2`, etc.
-      - Uses `.controlplane/Dockerfile`
+      - Uses `.controlplane/Dockerfile` or a different Dockerfile specified through `dockerfile` in the `.controlplane/controlplane.yml` file
+      - If a commit is provided through `--commit` or `-c`, it will be set as the runtime env var `GIT_COMMIT`
     DESC
 
-    def call
+    def call # rubocop:disable Metrics/MethodLength
       ensure_docker_running!
 
       dockerfile = config.current[:dockerfile] || "Dockerfile"
       dockerfile = "#{config.app_cpln_dir}/#{dockerfile}"
 
+      raise "Can't find Dockerfile at '#{dockerfile}'." unless File.exist?(dockerfile)
+
       progress.puts("Building image from Dockerfile '#{dockerfile}'...\n\n")
 
-      cp.image_build(latest_image_next, dockerfile: dockerfile)
```

Comment:
@rafaelgomesxyz should we remove `cp.image_build`?

Reply:
I wasn't sure if we wanted to keep it around to maybe use it for something else, but ok, removed it and renamed `image_build_with_docker` to `image_build`.

Reply:
why image_build and not build_image?

Reply:
Just keeping it consistent with Sergey's original function names, which are all in the `noun + verb` form. We might want to rename all methods to the `verb + noun` form, we just have to be consistent.